templates/patterns/modal/pattern-modal.html.twig

```diff
@@ -1,15 +1,13 @@
-{% set id = 'modal-' ~ random(1000) %}
-
-
-{% set attributes = attributes.addClass(['modal', 'fade']) %}
-{% if static_backdrop %}
-  {% set attributes = attributes.setAttribute('data-bs-backdrop', 'static') %}
-  {% set attributes = attributes.setAttribute('data-bs-keyboard', false) %}
-{% endif %}
+{#
+/**
+ * @file
+ * Modal component.
+ */
+#}
 
 {% block trigger %}
   {{ pattern('button', {
```

Comment:
what if we want to use this like we do now here https://github.com/openeuropa/oe_whitelabel/blob/1.x/templates/overrides/navigation/links--language-block.html.twig#L26 where the trigger should be a link, also we may want to use an existing link and have the modal printer somewhere else, so this is what I suggest:
lets wrap the button pattern in an if, ex `if btn_label is not empty` then the trigger can come from soemwhere else and we dont care what it is